mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -238,14 +238,16 @@
     <string name="search_suggestions_onboarding_title">Allow search suggestions in private sessions?</string>
     <!-- Search suggestion onboarding hint description text, first parameter is the name of the app defined in app_name (for example: Fenix)-->
     <string name="search_suggestions_onboarding_text">%s will share everything you type in the address bar with your default search engine.</string>
-    <!-- Search engine suggestion title text. The first parameter is the name of teh suggested engine-->
+    <!-- Search engine suggestion title text. The first parameter is the name of the suggested engine-->
     <string name="search_engine_suggestions_title">Search %s</string>
     <!-- Search engine suggestion description text -->
     <string name="search_engine_suggestions_description">Search directly from the address bar</string>
     <!-- Menu option in the search selector menu to open the search settings -->
     <string name="search_settings_menu_item">Search settings</string>
     <!-- Header text for the search selector menu -->
     <string name="search_header_menu_item_2">This time search in:</string>
+    <!-- Content description (not visible, for screen readers etc.): Search engine icon -->
+    <string name="search_engine_icon_content_description" tools:ignore="UnusedResources">%s search engine</string>
```

Comment:
Explain what the placeholder will be replaced by

Reply:
Thanks for catching that! It should have been:

<!-- Content description (not visible, for screen readers etc.): Search engine icon. The first parameter is the search engine name (for example: DuckDuckGo). -->

Reply:
Thanks @mavduevskiy. Can you let me know when the update lands today? I'll retrigger a string import immediately then, so that I can expose all these strings to localizers (there are quite a few strings here landing past string freeze so there's a risk to that). thanks

Reply:
@Delphine 
[Landed](https://github.com/mozilla-mobile/firefox-android/pull/2360)!